### PR TITLE
Added hw4

### PR DESCRIPTION
**Pull request by olenahor**
Hello, Kate. Please have a look on my hw4

### CODE REVIEW

File: horokhovska/hw4/task1.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            if (year % 400) == 0:  
+                print(year, "is a leap year") 
+            else:  
+                print(year, "is not a leap year")
+        else:  
+            print(year, "is a leap year")  
+    else:
+        print(year, "is not a leap year")
+
+if (year <= 99):
```
Comment (katemalash) [inline horokhovska/hw4/task1.py:17]: is this "if" necessary?

File: horokhovska/hw4/task1.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    print ('It is a first century')
+else:
+    print(year // 100 + 1, "century")
```
Comment (katemalash) [inline horokhovska/hw4/task1.py:20]: Good Decision, in general

File: horokhovska/hw4/task2.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+print("Please choose a figure by number to calculate a square: 1 - rectangle, 2 - triangle, 3 - circle")
```
Comment (katemalash) [inline horokhovska/hw4/task2.py:1]: This message could be passed as "input()" argument

File: horokhovska/hw4/task2.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    s = sqrt(p * (p - a) * (p - b) * (p - c))
+    print("The square of triangle is ", s)
+elif (usersChoise == 3):
+    print("You have choosed a cirle, please enter the radius: ")
+    r = float(input("r = "))
+    from math import pi
+    print("The square of circle is ", pi*r*r)
+else:
+    print("Wrong number")
+
```
Comment (katemalash) [inline horokhovska/hw4/task2.py:25]: Please remove this empty line

File: horokhovska/hw4/task3.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+number =int(input("Pleaase enter the number: "))
+if (number > 0):
+    a="The number a positive and"
+else:
+    a="The number is negative and"
+if (number < 10 and number > -10):
+    b="one-digit"
+else:
+    b="two-digit"
+print(a, b)
```
Comment (katemalash) [inline horokhovska/hw4/task3.py:10]: This decision works but what about, for example, "100500"?

File: horokhovska/hw4/task4.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        print(c, "x 10 UAH")
+else:
+        a=money//200
+        b=(money-a*200)//100
+        c=(money-a*200-b*100)//10
+        d=money-a*200-b*100-c*10
+        print(a, "x 200 UAH")
+        print(b, "x 100 UAH")
+        print(c, "x 10 UAH")
+        print(d, "x 1 UAH")
```
Comment (katemalash) [inline horokhovska/hw4/task4.py:25]: Good Decision

File: horokhovska/hw4/task5.py [hunk 7]
```diff
@@ -0,0 +1,8 @@
+r = float(input("Please enter a stage radius: "))
+s = float(input("Please enter a hall square: "))
+k = float(input("Please enter a distance from the wall to the stage: "))
+
+from math import sqrt
+a = sqrt(s)
+k = (a-2*r)/2
+print("The distance from the wall to the stage is: ", k)
```
Comment (katemalash) [inline horokhovska/hw4/task5.py:8]: It was required to input K but not define it

File: horokhovska/hw4/task6.py [hunk 8]
```diff
@@ -0,0 +1,14 @@
+seat_number = int(input("Please enter a seat number: "))
+
+if seat_number >= 1 and seat_number <= 36:
```
Comment (katemalash) [inline horokhovska/hw4/task6.py:3]: Just clarification: this option also works `1 <= seat_number <= 36`

File: horokhovska/hw4/task6.py [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+elif seat_number > 36 and seat_number <= 53: 
+        a="Your sear is side and"
+else: 
+        print("There is only 53 seats in the carriage. Try again")
+
+if seat_number // 2 == 0:
+        b="on the top"
+else:
+        b="on the bottom"
+print(a,b)
```
Comment (katemalash) [inline horokhovska/hw4/task6.py:14]: What will be printed if user enters "60"?

**Review by katemalash — APPROVED** [review]
Good, in general. You can merge this pull-request after fixing some issues.

File: horokhovska/hw4/task1.py (hunk 1 above)
Comment (olenahor) [reply horokhovska/hw4/task1.py:17]: which one?

Comment (katemalash) [reply horokhovska/hw4/task1.py:17]: in line 17

**Review by katemalash — COMMENTED** [review]
Looks nice now.

**Pull request merged**